websites/docs-smoke-test/src/content/components/video-player.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: VideoPlayer component
+---
+
+# Video Player
+
+<VideoPlayer options={{
```

Comment:
My "wishlist" Author interface would be: 

```jsx
<Video 
   url="https://bitdash-a.akamaihd.net/content/sintel/hls/playlist.m3u8" 
   posterPath="foobar-module/intro-video-poster.jpg"  // a path to an image in the repo following the same conventions as when we refer to other images 
/> 
```
- No autoplay (does not work reliable anyways and browser vendors do not want it) https://videojs.com/blog/autoplay-best-practices-with-video-js/
-  just "video" 
- No low level options passthrough (see separate comment)
- "Poster" describes it better,  a "thumbnail" is a small size version of a bigger image. That's also the name in the HTML standard https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video

Reply:
stupid question: Is the GitHub octocat the default poster image inside videojs or did you set that or is it in the playlist you linked?     In any case - it's not exactly ideal as a default (other peoples' brand).

Reply:
No, it was just a test since locally the static images are not working, so I wanted and absolute url. I'll replace it with a commercetools logo